src/engine/strat_engine/pool.rs

```diff
@@ -41,12 +41,12 @@ use super::metadata::MIN_MDA_SECTORS;
 use super::serde_structs::{FilesystemSave, FlexDevsSave, PoolSave, Recordable, ThinPoolDevSave};
 use super::setup::{get_blockdevs, get_metadata};
 
-const DATA_BLOCK_SIZE: Sectors = Sectors(2048);
+pub const DATA_BLOCK_SIZE: Sectors = Sectors(2048);
 const META_LOWATER: u64 = 512;
 const DATA_LOWATER: DataBlocks = DataBlocks(512);
-
+const DATA_EXPANSION_PERCENT: f32 = 20.0;
```

Comment:
I was thinking that slow growth would make things easier on a storage admin.  I think doubling or moving to the next power of 2 is fine for very small filesystems, but I don't think it scales.  I think in the long run - we will want to make this a modifiable setting.  How about if I put it in a function, open an issue and we can discuss when @agrover gets back?